src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -931,5 +932,107 @@ public void WorkingDirectory_Set_GetReturnsExpected(string workingDirectory)
             var info = new ProcessStartInfo { WorkingDirectory = workingDirectory };
             Assert.Equal(workingDirectory ?? string.Empty, info.WorkingDirectory);
         }
+
+        [Fact(Skip = "Manual test")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void StartInfo_WebPage()
+        {
+            ProcessStartInfo info = new ProcessStartInfo
```

Comment:
We've done this in some test suites with env vars and a ConditionalFact that checks for its presence.  Then someone working on the library can set that env var and run the test suite, but in CI and by default locally it won't run.